concepts/decorators/about.md

```diff
@@ -1,2 +1,44 @@
-#TODO: Add about for this concept.
+# About
 
+Decorators are a feature of the Python language that can modify or register a function.
+
+## How to use them
+
+Decorators are placed just above a function in the code like so:
+
+```py
+@decorator
+def function():
+    pass
+```
+
+Some decorator functions take arguments:
+
+```py
+@decorator2(name="Bob")
+def function2():
+    pass
+```
+
+If a decorator function takes any arguments, but you don't wish to pass any, you must still call it with parentheses for the decorator to work:
+
+```py
+@decorator2()
+def function3():
+    pass
+```
+
+Decorators are just syntactic sugar. An alternative way of doing the same as the above three examples is using higher-order functions:
+```py
+def function():
+    pass
+function = decorator(function)
+
+def function2():
+    pass
+function2 = decorator2(name="Bob")(function2)
+
+def function3():
+    pass
+function3 = decorator2()(function3)
+```
```

Comment:
Technical point, missing EOL (end of line) character at line 44.

Reply:
I'm not quite sure what you mean?

Reply:
It looks like you fixed it, by appending `\r\n` to the end of the file.  But what I mean is that by definition lines have an end of line, and it is expected.  Sometimes our tools seem to forget this and because this is a text file, and text files have lines, this one character is expected at the end of each line.